src/components/report/VariableControls.vue

```diff
@@ -101,14 +104,39 @@
 				:value="activeInstance"
 			>
 				<option
-					v-for="(axes, instance) in instances"
+					v-for="(_, instance) in instances"
 					:key="instance"
 					:selected="instance === activeInstance"
 				>
 					{{ instance }}
 				</option>
 			</select>
+			<button
+				type="button"
+				class="button"
+				v-if="showPreviewButton && showPreviews"
+				@click="showPreviews = false"
+			>
+				Hide previews
+			</button>
+			<button
+				type="button"
+				class="button"
+				v-if="showPreviewButton && !showPreviews"
+				@click="showPreviews = true"
+			>
+				Preview all
+			</button>
```

Comment:
Alternative:

```patch
- 		<button
- 				type="button"
- 				class="button"
- 				v-if="showPreviewButton && showPreviews"
- 				@click="showPreviews = false"
- 			>
- 				Hide previews
- 			</button>
- 			<button
- 				type="button"
- 				class="button"
- 				v-if="showPreviewButton && !showPreviews"
- 				@click="showPreviews = true"
- 			>
- 				Preview all
- 			</button>
+ 		<button
+ 				type="button"
+ 				class="button"
+ 				v-if="showPreviewButton"
+ 				@click="showPreviews = !showPreviews"
+ 			>
+ 				{{ showPreviews ? 'Hide previews' : 'Preview all' }}
+ 			</button>
```
(untested)

Reply:
I continue sharing these kind of suggestions in case you want to apply them somewhere else. Think of them more as a sounding board than a request to change.